backend/src/api/types/requests/FeedSortingSave.ts

```diff
@@ -0,0 +1,8 @@
+import {FeedSorting} from '../../../db/types/FeedSortingSettings';
```

Comment:
Don't use non-API types in API types.
You should duplicate type or you can do reverse thing – use API type in non-API types.

Reason for this – I want to extract API types declarations into separate library to reuse it in backend and frontend.